js/model/crm.uf.js

```diff
@@ -358,12 +358,14 @@
      *
      */
     markDuplicates: function() {
-      var ufFieldModelsByKey = this.groupBy(function(ufFieldModel) {
+      const ufFieldModelsByKey = this.groupBy(function(ufFieldModel) {
         return ufFieldModel.getSignature();
       });
       this.each(function(ufFieldModel){
-        var is_duplicate = ufFieldModelsByKey[ufFieldModel.getSignature()].length > 1;
-        if (is_duplicate != ufFieldModel.get('is_duplicate')) {
+        const length = ufFieldModelsByKey[ufFieldModel.getSignature()].length > 1;
+        const label = ufFieldModelsByKey[ufFieldModel.getSignature()][0].attributes.label !== 'Free HTML';
```

Comment:
This would only work on an english site, or even on an english site only works if you don't change the label, but that's not a blocker since it doesn't break anything. But this whole set of code looks like it will be replaced in future: https://lab.civicrm.org/dev/core/-/issues/4349

Also these variable names are a bit non-standard, i.e. length and label are referring to boolean values, but again this code will probably be replaced.

Having said that, it's mergeable as-is, and if the replacement is a long way off then it's worth putting this in now.

Reply:
jenkins retest this please